tests/index.js

```diff
@@ -86,17 +84,14 @@ describe('HPKP Headers', function () {
       method: "GET",
       url: "/"
     }
-    before(function () {
-      server = createServer(3000, testCase.options)
-    })
 
-    after(function () {
-      return server.stop()
-    })
-    it(testCase.name, function (done) {
-      server.inject(requestOptions, function (response) {
-        assert.equal(response.headers[testCase.expectedKey], testCase.expectedHeader)
-        done()
+    it(testCase.name, function () {
+      return createServer(3000, testCase.options).then((s) => {
```

Comment:
@deeptibaghel before merging this let's clean this up to maybe use `before/after` again. More importantly let's see if this plugin works in the fxa-auth-server before merging :)

Reply:
Still applies, let's clean this up